v2/mfa/pre-built-ui/protecting-routes.mdx

```diff
@@ -76,8 +79,8 @@ function App() {
         <SuperTokensWrapper>
             <div className="App">
                 <div className="fill">
-                    <Routes>
-                        {getSuperTokensRoutesForReactRouterDom(reactRouterDom)}
+                    <Routes> 
+                        {getSuperTokensRoutesForReactRouterDom(reactRouterDom, [EmailPasswordPreBuiltUI, PasswordlessPreBuiltUI])}
```

Comment:
I think we are using ThirdPartyEmailPassword in mfa docs. So change emailpassword to thirdpartyemailpassword in all the mfa docs